docker-compose example/docker-compose.example.yml

```diff
@@ -42,6 +42,10 @@ services:
       ## A token can be generated for free here https://account.mapbox.com/access-tokens/
       ## It's a good idea to limit the scope of the token to your own domain, to prevent others from using it.
       MAPBOX_TOKEN: ${MAPBOX_TOKEN}
+      ## Optional: Set the hardware acceleration when encoding videos.
+      ## Support `qsv`, `vaapi`, `nvenc`.
+      ## Only `qsv` is verified with `/dev/dri` devices (see below `devices`).
+      # PHOTOVIEW_VIDEO_HARDWARE_ACCELERATION=
```

Comment:
If a var is part of the .env, it should be just transparently mapped in the compose, not assigned. 
So, instead of this line:
`# PHOTOVIEW_VIDEO_HARDWARE_ACCELERATION: ${PHOTOVIEW_VIDEO_HARDWARE_ACCELERATION}`
In the same way as the MAPBOX_TOKEN (or another var) is mapped

Reply:
Done.